src/features/machine/computer.js

```diff
@@ -47,20 +48,42 @@ export const transformShape = (data, vertex, amount, trackIndex=0, numLoops) =>
   let transformedVertex = vertex
 
   if (data.repeatEnabled && data.growEnabled) {
-    transformedVertex = scale(transformedVertex, 100.0 + (data.growValue * amount))
+    var growAmount = 100
+    if (data.growAdvanced) {
+      try {
+        growAmount = data.growValue * evaluate(data.growMath, {i: amount})
+      }
+      catch (err) {
+        console.log(err)
```

Comment:
If you are leaving this for tech-savvy users, you might prefix it with "error parsing grow function" (similar for spin). If you aren't, you might comment these out.